src/Microsoft.ML.CpuMath/Microsoft.ML.CpuMath.csproj

```diff
@@ -16,7 +15,16 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
+    <Compile Remove="CpuMathUtils.DotNetStandard.cs" />
```

Comment:
Let's use the normal TFM naming and casing on this file suffix, that's how we do it in corefx:

`CpuMathUtils.netstandard.cs`
`CpuMathUtils.netcoreapp.cs`